### DISCUSSION TITLE
File box data clarifications

### DISCUSSION THREAD

**Discussion by Blu3train** [buliasz/tesstrain-windows-gui#25] [Q&A] [answered]
I edited the box files created by the trainer and I noticed that each character box has the size of the entire image as a box.
I have the same result with both tesstrain-windows-gui and my makefile that I used before.

I had never edited these files, but since reading the various tutorials I saw that one of the operations I should do is to verify that the single letter has the box that surrounds it, I was perplexed.

I tried to do both English and Japanese training, and the result is identical.
I am attaching an example of an image file and box.

Is it correct that this is so? What am I doing wrong?
[0en001.zip](https://github.com/buliasz/tesstrain-windows-gui/files/9627947/0en001.zip)

**Comment by buliasz** [comment, marked as answer]
I think [this documentation page](https://tesseract-ocr.github.io/tessdoc/tess4/Make-Box-Files.html) from the official Tesseract site should answer your question. The new LSTM training just uses whole lines, so the old per-character boxes are no longer necessary.

In other words you are doing it right and it is expected.

**Reply by Blu3train** [reply]
You are very kind. I really thank you very much
